examples/player/simid_player.html

```diff
@@ -23,6 +23,20 @@
     }
   </style>
   <script>
+    //if ad is non-linear allow user to insert specific dimensions
+    function enableDisable() {
```

Comment:
Maybe call the function something like.

modifyControlVisibility

Reply:
this function is called setButtonState (it's changed in the buttons PR)